src/HealthGPS/energy_balance_hierarchical_model.cpp

```diff
@@ -37,15 +46,29 @@ void EnergyBalanceHierarchicalModel::update_risk_factors(RuntimeContext &context
             current_risk_factors.at(age_key) = model_age;
         }
 
-        auto &equations = definition_.get().at(model_age);
+        auto &eqns = equations_at(model_age);
```

Comment:
Why do you shorten this to `eqns`? Writing `equations` in full seems more readable.

Reply:
Not sure why that happened. I can change it back.